app/views/pensions/lifetimeAllowance/PensionProviderPaidTaxView.scala.html

```diff
@@ -39,7 +39,7 @@
 @(form: Form[(Boolean, Option[BigDecimal])], taxYear: Int)(implicit request: AuthorisationRequest[_], messages: Messages, appConfig: AppConfig)
 
 @titleForUse = @{
-    messages(s"pensions.pensionProviderPaidTax.title.${if(request.user.isAgent) "agent" else "individual"}")
+    messages(s"common.pensions.didYourPensionSchemesAgreeToPayTheTax.title.${if(request.user.isAgent) "agent" else "individual"}")
```

Comment:
This page doesn't seem to match this part of the message key: `didYourPensionSchemesAgreeToPayTheTax`.

Reply:
Sorry I dont understand this comment

Reply:
This view refers to "pension provider paid tax"; but part of the message key seems to refer to a different page.

Reply:
yes that's because this message is generic (that's why in common). So I felt the key has to reflect what the message  is

Reply:
OK. I thought it was a page.